feature-libs/quote/core/connectors/quote.connector.ts

```diff
@@ -6,73 +6,124 @@
 
 import { inject, Injectable } from '@angular/core';
 import { PaginationModel } from '@spartacus/core';
-import { Observable } from 'rxjs';
 import {
+  Comment,
   Quote,
   QuoteActionType,
   QuoteDiscount,
   QuoteList,
   QuoteMetadata,
   QuoteStarter,
-  Comment,
 } from '@spartacus/quote/root';
+import { Observable } from 'rxjs';
 import { QuoteAdapter } from './quote.adapter';
 
 @Injectable()
 export class QuoteConnector {
   protected quoteAdapter = inject(QuoteAdapter);
 
-  public getQuotes(
+  /**
+   * Fetches a list of quotes according to the pagination provided
```

Comment:
Could we use `retrieves` instead of `fetches / gets` ...?

Reply:
No, I guess 'fetch' is even better, e.g. check https://thecontentauthority.com/blog/fetch-vs-retrieve